src/NamingStrategy/PathNamingStrategy.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Http\Client\Plugin\Vcr\NamingStrategy;
+
+use Psr\Http\Message\RequestInterface;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Will use the request path as filename.
```

Comment:
looking at the code, this actually does more than use the path. it also uses hostname, query string and request method. do we really need an interface? what other strategy could be used?

Reply:
Nothing specific in mind, but some might use the Symfony Router for example, and use the route name as filename, or use some specific business logic.

Reply:
fair enough. then lets just adjust the phpdoc here to not give the impression its only about path.